decidim-core/spec/tasks/decidim_tasks_taxonomies_spec.rb

```diff
@@ -5,7 +5,13 @@
 
 describe "Executing Decidim Taxonomy importer tasks" do
   let(:plan_file) { Rails.root.join("tmp/taxonomies/#{organization.host}_plan.json") }
-  let(:organization) { create(:organization) }
+  let(:another_plan_file) { Rails.root.join("tmp/taxonomies/#{another_organization.host}_plan.json") }
+
+  let(:organization) { create(:organization, host: "billie.example.org") }
+  let(:another_organization) { create(:organization, host: "ella.example.org") }
```

Comment:
Checking out other examples, this is how we've done it:

```suggestion
  let(:organization) { create(:organization, host: "foo.example.org") }
  let(:another_organization) { create(:organization, host: "bar.example.org") }
```